### PR TITLE
HDDS-9302. Summarize failures of acceptance/kubernetes checks

### PR DESCRIPTION
**Pull request by adoroszlai**
## What changes were proposed in this pull request?

Acceptance and kubernetes CI checks provide HTML-based report from Robot as summary, which is great for checking locally.  This PR adds a text-based summary so that we can give a quick overview in the Github Actions UI.

https://issues.apache.org/jira/browse/HDDS-9302

## How was this patch tested?

Tested in CI.

Failed acceptance run:
https://github.com/adoroszlai/hadoop-ozone/actions/runs/6224728553/job/16894469092#step:6:1

Successful acceptance run:
https://github.com/adoroszlai/hadoop-ozone/actions/runs/6224728553/job/16894468903#step:6:1

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -306,6 +309,9 @@ jobs:
           sudo mkdir .aws && sudo chmod 777 .aws && sudo chown 1000 .aws
           popd
           ./hadoop-ozone/dev-support/checks/kubernetes.sh
+      - name: Summary of failures
+        run: cat target/${{ github.job }}/summary.txt
+        if: ${{ !cancelled() }}
```
Comment (aswinshakil) [inline .github/workflows/ci.yml:314]: I have a quick question, why is it that for integration test we use
```
      - name: Summary of failures
        run: cat target/${{ github.job }}/summary.txt
        if: always()
```
Shouldn't that also be only if the job is not canceled?

Comment (adoroszlai) [reply .github/workflows/ci.yml:314]: Right, it should be.

Comment (adoroszlai) [reply .github/workflows/ci.yml:355]: Thanks @aswinshakil for pointing this out.  Patch is updated, can you please take another look?

**Review by aswinshakil — APPROVED** [review]
Thanks for the CI improvements @adoroszlai. LGTM!

**Comment by adoroszlai** [comment]
Thanks @aswinshakil for the review.

**Pull request merged**